.github/workflows/pr-check.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Check Type
         uses: gsactions/commit-message-checker@v2
         with:
-          pattern: '^(build|ci|docs|feat|fix|refactor|test)(|!): [a-z].+?((, fixes #\d+)*)($|\n|\r|\n\r)'
+          pattern: '^(build|ci|docs|feat|fix|refactor|test)(|!): [A-Za-z].+?((, fixes #\d+)*)($|\n|\r|\n\r)'
```

Comment:
Looks right to me. Can you test it on your fork?

Reply:
Don't forget to change the docs with this.

Reply:
Well, the docs already match this pattern since they ask for the first letter to be an upperacase.
It would be tricky to test it maybe as a post-commit hook but that be a bit different.

Reply:
Oh, didn't even realize that was the reason for your other PR.